weblate/addons/base.py

```diff
@@ -115,16 +116,21 @@ def configure(self, settings):
         self.post_configure()
 
     def post_configure(self, run: bool = True):
-        component = self.instance.component
+        target = self.instance.component or self.instance.project
 
         # Configure events to current status
-        component.log_debug("configuring events for %s add-on", self.name)
+        target.log_debug("configuring events for %s add-on", self.name)
         self.instance.configure_events(self.events)
 
         if run:
-            postconfigure_addon.delay(self.instance.pk)
-            # Flush cache in case this was eager mode
-            component.repository.clean_revision_cache()
+            if isinstance(target, Project):
+                components = target.component_set.all()
+                for component in components:
+                    postconfigure_addon.delay(component.pk)
+                    component.repository.clean_revision_cache()
+            else:
+                postconfigure_addon.delay(target.pk)
+                component.repository.clean_revision_cache()
```

Comment:
`postconfigure_addon` received AddOn instance, not a component instance, so the loop is not needed here. I'm rewriting this code anyway in https://github.com/WeblateOrg/weblate/pull/10149:

```suggestion
            postconfigure_addon.delay(self.instance.pk)
            # Flush cache in case this was eager mode
            component.repository.clean_revision_cache()
```